### DISCUSSION TITLE
Timoni support

### DISCUSSION THREAD

**Discussion by pmig** [glasskube/glasskube#242] [Ideas]
Timoni bundles could be supported in similar way Glasskube supports Helm charts.

Is this something you are interested in?

**Comment by luolong** [comment]
Yes, very much so

**Reply by jamroks** [reply]
Yes indeed, that would wonderful, 🙌🏽🚀

**Comment by jamroks** [comment]
Yes that would be wonderful, the Timoni support would greatly ease our adoption of glasskube, since we have heavily invest in Timoni. Best of both world 🚀😍
